pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -246,7 +259,10 @@ public boolean isHelp() {
     @Parameter(names = { "-h", "--help" }, description = "Show this help message")
     private boolean help = false;
 
+    private boolean usingNew211Defaults;
+
     public void start() throws Exception {
+        usingNew211Defaults = !Paths.get(zkDir).toFile().exists();
```

Comment:
Os this is true maybe we could log a line explaining that we found  the directory and we are switching to the old defaults

Reply:
A warning log would be great when both `--metadata-dir` is provided and `zkDir` exists.